source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -329,6 +336,27 @@ export default class NetworkStatusStore extends Store {
 
   // DEFINE ACTIONS
 
+  @action updateInternetConnectionStatus = async () => {
+    try {
+      await externalRequest(
+        {
+          hostname: CHECK_INTERNET_PING_HOSTNAME,
```

Comment:
@yakovkaravelov something like `CHECK_INTERNET_CONNECTION_HOSTNAME ` is more descriptive.